tests/integration/pilot/vm/testdata/OS_Support_List

```diff
@@ -0,0 +1,6 @@
+app_sidecar_trusty
```

Comment:
Why do we need a file? instead of just defining a list

Reply:
Updated to use a defined list instead.